src/sales/sales-menu.md

```diff
@@ -22,45 +22,45 @@ On the _Admin_ sidebar, click **Sales**.
 ## Menu Options
 
 <!--{% if "Default.B2B Only" contains site.edition %}-->
-### [Quotes]({{ site.baseurl }}{% link sales/quotes.md %})
+### Quotes
 
 Authorized buyers can negotiation the price with the seller by sending a [request]({{ site.baseurl }}{% link sales/quote-price-negotiation.md %}) from the shopping cart.
```

Comment:
Find a place to link the `({{ site.baseurl }}{% link sales/quotes.md %})` page.

Reply:
**done**